lib/generators/hello/locales/USAGE

```diff
@@ -0,0 +1,8 @@
+Description:
+  This generators copy available locales.
+
+Example:
+    rails generate hello:locales en
+
+    This will create:
+        config/locales/hello.en.yml
```

Comment:
how about we add some usage instructions?
 +
 +    rails generate hello:locales
 +
 +    This will show you all locales